### ISSUE TITLE
"Failed to parse source map" warnings

### ISSUE THREAD

**Issue by codebutler** [kruschid/typesafe-routes#25]
Seeing this in the console when starting a local dev server (webpack):

```
WARNING in ./node_modules/typesafe-routes/build/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/index.ts' file: Error: ENOENT: no such file or directory, open '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/index.ts'

WARNING in ./node_modules/typesafe-routes/build/parser.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/parser.ts' file: Error: ENOENT: no such file or directory, open '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/parser.ts'

WARNING in ./node_modules/typesafe-routes/build/react-router.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/react-router.tsx' file: Error: ENOENT: no such file or directory, open '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/react-router.tsx'

WARNING in ./node_modules/typesafe-routes/build/route.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/route.ts' file: Error: ENOENT: no such file or directory, open '/Users/eric/Code/frontend-react/node_modules/typesafe-routes/src/route.ts'
```

**Comment by kruschid** [comment]
415860d9c3eedae24c59a57c7ca6a50afdf394f7 should fix that. It's published under the version `9.0.1`.

**Comment by kruschid** [comment]
should better add some build automation soon. :-)

Thank you for reporting
